docs/_clef/CliqueSigning.md

```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
 INFO [06-16|11:14:54.125] Successfully wrote genesis state         database=lightchaindata hash=187412..4deb98
 ```
 
-At this point, we have a Clique network which we can start sealing on.
+At this point a Clique network exists with blocks that can be signed.
```

Comment:
```suggestion
At this point a Clique network exists with blocks that can be signed.
```
Well, we've initiated geth with a geneiss, but no blocks exist yet.

Reply:
yes ofc - thanks!

```suggestion
At this point a Geth has been initiated with a genesis configuration.
```